specs/cli/cli.md

```diff
@@ -0,0 +1,55 @@
+# FAST CLI (WIP)
+
+## Overview
+FAST CLI is a set of tools used for creating, building and developing FAST projects.
+
+FAST CLI will have an independent code base and will be packaged and installable on any operating system.
+
+## Use Cases
+- A user can use FAST CLI to generate a new FAST design system project with a single command.
+- A user can generate a new component file structure in FAST using a single command in FAST CLI
+
+## Features
+- Project generator based on given name string
+- Component generator
+- Cross-platform
```

Comment:
For improved developer efficiency, I'd prefer to not require installing additional dependencies to reach cross-platform capability.